utilities/account/integration/ohada.csv

```diff
@@ -0,0 +1,1193 @@
+,70611018,Accouchement PAX
+,70611019,Imagerie Medicale
+,70611020,Orl
+,70611021,Perfusion
+,70611022,Acouchement
+,70611023,Soins infirmiers ambulatoires
+,70611024,Greffes dermo-epidermiques
+,70611025,Excisions
+,70611026,Incisions
+,70611027,Orthopedie
+,70611028,Gynecologie
+,70611029,Examen complementaire
+,70611030,Laparatomie
+,70611031,Obstetrique
+,70611032,Traumatologie fractures
+,70611033,Petite chirurgie sutures
+,70611034,Urologie
+,70611035,Stomatologie
+,70611036,URGENCES
+,71,71 SUBVENTIONS D'EXPLOITATION
+,718,AUTRES SUBVENTIONS D'EXPLOITATION
+,71811010,Autres subenventions d'expl. Versees par l'�tat et les collectivites publiques
+,7182,Autres subenventions d'expl.Versees par les organismes internationaux
+,71821010,Versees par WOMEN'S DEVELOPMENT
+,71821011,Versees par AGRICULTURAL DEVELOPMENT
+,71821012,Versees par NUTRITION REAB. (CN)
+,71821013,Versees par INDIGENT CARE PROJECT
+,71821014,Versees par TSHIKAJI HEALTH CENTER
+,71821015,Versees par ORPHAN SCHOR EDUC PROJECT
+,71821016,Versees par HMHC
+,71821017,Versees par FISTULA PROJECT
+,71821018,Versees par CERVICAL CANCER
+,71821019,Versees par BCZS COM. HEALTH PROGRAM
+,71821020,Versees par MOBILITY IN MISSION 0096AA-08
+,71821021,Versees par SUPPORT OF THE NURSING SCHOOL
+,71821022,Versees par WOMEN DEVELOPMENT
+,71821023,Versees par ORPHAN SCHOLARSHIP
+,71821024,Versees par MOTHER CARE BABY CARE
+,71821025,Versees par C-GSH FISTULA REPAIR
+,71821026,Versees par FOUNDERS CHAPEL
+,71821027,Versees par MISSIONNARY HOUSING
+,71821028,Versees par CHARITY FUNDS
+,7183,Autres subenventions d'expl. Versees par des tiers
+,71831010,Versees par KAJIDI JONATHAN SCHOLARSHIP
+,72,PRODUCTION IMMOBILIS�E
+,721,PRODUCTION IMMOBILIS�E - IMMOBILISATIONS INCORPORELLES
+,72111010,Production Immobilisee-Immobilisations Incorporelles
+,722,PRODUCTION IMMOBILIS�E - IMMOBILISATIONS CORPORELLES
+,72211010,Production Immobilisee-Immobilisations Corporelles
+,73,VARIATIONS DES STOCKS DE BIENS ET DE SERVICES PRODUITS
+,736,Variation des Stocks des Produits Finis
+,73611010,Variation des Produits de Perfusion
+,73611011,Variation des Produits Ophtamologiques
+,75,AUTRES PRODUITS
+,754,PRODUITS DES CESSIONS D'IMMOBILISATIONS
+,75411010,Produits des Cessions d'Immobilisations
+,758,PRODUITS DIVERS
+,7581,Jetons de presence et autres remunerations d'administrateurs
+,75811010,Jeton de presence
+,75811011,Autres remunerations d'administrateurs
+,7582,Indemnites d�assurances re�ues
+,75821010,Indemnites d�assurances re�ues
+,7588,Autres Produits divers
+,75881010,Autres revenus
+,759,REPRISES DE CHARGES PROVISIONN�ES D'EXPLOITATION
+,7593,Reprises de charges provisionnees sur stocks
+,75931010,Reprise des charges provisionnees pour stocks obsol�te
+,7594,Reprises de charges provisionnees sur creances
+,75941010,Reprise de charges provisionnees sur creances impayees
+,7598,Reprises sur autres charges provisionnees
+,77,REVENUS FINANCIERS ET PRODUITS ASSIMIL�S
+,771,INT�R�TS DE PR�TS

```

Comment:
Is there any way to remove the code points that are causing these ? to appear?  They will probably not be inserted correctly into the database.